MUDscape/driver.py

```diff
@@ -73,6 +82,22 @@ def send(self, message='', prompt=True):
 
         self.request.sendall(message.format(**Fore.__dict__).encode('utf-8'))
 
+    def bufferInput(self):
+        data = bytes()
+        newLine = "b\'\\r\\n\'"
```

Comment:
I wasn't sure how to do it, so I used a hacky type casting way instead. will update to the proper way.